### PR TITLE
Show engine in Task trouble shooting logs

### PR DESCRIPTION
**Pull request by dpsutton**
## Including database engine

image

---
#### When no database:

image

in an effort to get better feedback when people raise issues about
long running sync processes, include the database engine in the logs.

Also wrapped `th` in a `tr` inside the `thead` to remove react warning

### CODE REVIEW

**Comment by dpsutton** [comment]
things would love feedback on:
- what to put when there's no db_id and therefore no engine: is null fine, a dash, "N/A", don't care... etc
- stylistically, i didn't know what to name the `id->engine` map. i went with `db_id_to_engine`. In clojure it would have been `id->engine` which is nice and pretty clear. The underscores and `to` leave a lot to be desired IMO.

small quibbles but interested

File: frontend/src/metabase/admin/tasks/containers/TasksApp.jsx [hunk 1]
```diff
@@ -12,16 +13,22 @@ import Tooltip from "metabase/components/Tooltip";
...
       tasks,
+      databases,
       page,
       pageSize,
       onNextPage,
       onPreviousPage,
       children,
     } = this.props;
+    const db_id_to_engine = {};
+    for (const db of databases) {
```
Comment (kdoh) [inline frontend/src/metabase/admin/tasks/containers/TasksApp.jsx:29]: For what it's doing, I don't think it's worth moving this to an outside function or putting it inside `connect` or anything, but I might just leave a little note about what's being done here and why so it stands out a bit more and it's easier to grok on subsequent viewing.

**Review by kdoh — CHANGES_REQUESTED** [review]
Looking pretty good. Mostly just a few style related things on the code side related to naming and so forth.

From the non code side, I left a note about possibly replacing the database ID with the name, I think the ID is basically not helpful. As far as your question about what to put if there's no relevant engine for a task, I think having nothing in the cell is ok.

File: frontend/src/metabase/admin/tasks/containers/TasksApp.jsx [hunk 2]
```diff
@@ -53,18 +60,26 @@ class TasksApp extends React.Component {
...
+            </tr>
           </thead>
           <tbody>
             {tasks.map(task => (
               <tr key={task.id}>
                 <td className="text-bold">{task.task}</td>
                 <td>{task.db_id}</td>
+                <td>
+                  {task.db_id
+                    ? db_id_to_engine[task.db_id] || "Unknown engine"
```
[the remark this answers is not in the document]
Comment (dpsutton) [reply frontend/src/metabase/admin/tasks/containers/TasksApp.jsx:80]: @kdoh when i make this change is there some process to make sure this phrase gets slated for translation or is that automatically handled with tooling?

**Comment by dpsutton** [comment]
@kdoh can you approve the changes i made in response to your comments? And i'm assuming the failing tests are general flakiness not due to my changes. I tried to rerun from failed but got the same failing result.

**Review by kdoh — APPROVED** [review]

**Pull request merged**